documentation/manifest.md

```diff
@@ -93,9 +93,9 @@
         - [ ] Verify Issues button navigates to Issues page on GitHub.com (when logged in and the repository is enabled)
 - [ ] **Creating Multiple Repositories**
     - [ ] Go to Team Explorer Connect page (logged in)
-    - [ ] Create additional repository
-    - [ ] Publish new repository
-    - [ ] Verify remote information present in repository settings
+      - [ ] Create additional repository
+        - [ ] Publish new repository
+          - [ ] Verify remote information present in repository settings
```

Comment:
Did you mean to add the indentation here? I don't care, just checking whether it's intentional 😄